k8s/manifest-templates/pharos.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pharos
+  labels:
+    name: pharos
+spec:
+  hostNetwork: true
+  containers:
+  - name: pharos-dashboard
+    image: ereslibre/pharos-dev:latest
```

Comment:
Yes I know. I prefer to explicitly pull the image with ```docker pull``` when needed. Otherwise we must always keep this ```latest``` anomaly in mind.